src/redux/configureStore.js

```diff
@@ -0,0 +1,17 @@
+// configureStore.js
+
+import { createStore, combineReducers, applyMiddleware } from 'redux';
+import logger from 'redux-logger';
+import booksReducer from './Books/books';
```

Comment:
- [ ] Hi @sangwa7, also note that you were supposed to have added a reducer for Categories and import it here as the requirements demand. Please fix this issue.